pycbc/detector/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .ground import *
```

Comment:
If you do an import *, make sure the functions / classes that should be imported are included in an __all__ property at the bottom of the file. That way it only imports the things we actually want exposed at the higher level.